### ISSUE TITLE
Throwing an error in Meteor method reports "TypeError: stack.startsWith is not a function" on the console

### ISSUE THREAD

**Issue by SimonSimCity** [meteor/meteor#11001]
We currently see the following message on our console if a syntax-error is thrown by node inside a method:

```
(node:1) UnhandledPromiseRejectionWarning: TypeError: stack.startsWith is not a function
    at formatError (internal/util/inspect.js:915:13)
    at formatRaw (internal/util/inspect.js:703:14)
    at formatValue (internal/util/inspect.js:591:10)
    at inspect (internal/util/inspect.js:221:10)
    at formatWithOptions (internal/util/inspect.js:1693:40)
    at Object.Console.<computed> (internal/console/constructor.js:272:10)
    at Object.log (internal/console/constructor.js:282:61)
    at Object.Meteor._debug (packages/meteor.js:1055:23)
    at wrapInternalException (packages/ddp-server/livedata_server.js:1741:12)
    at packages/ddp-server/livedata_server.js:745:25
    at /opt/bundle/bundle/programs/server/npm/node_modules/meteor/promise/node_modules/meteor-promise/fiber_pool.js:43:40
UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
[DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

It seems to be related to https://github.com/nodejs/node/issues/30572 which is a node internal error - which should've been resolved in Node `v12.14.1`, according to the changelog - https://github.com/nodejs/node/blob/master/doc/changelogs/CHANGELOG_V12.md#2020-01-07-version-12141-erbium-lts-bethgriggs

I'm on Meteor `1.9.0` which includes Node `v12.14.0` and according to the release notes the version of Node wasn't changed lately.

**Comment by SimonSimCity** [comment]
Fixed by updating to Meteor `1.10.1` which - despite not being mentioned - updates NodeJS to `v12.16.1`.

**Comment by SimonSimCity** [comment]
I have to be fair - the update is mentioned as part of the Meteor release `1.9.1` and `1.9.2` ... I just haven't been awake enough to see it 🙈
